index.html

```diff
@@ -808,6 +811,90 @@ <h3>
           </p>
         </section>
       </section>
+      <section>
+        <h3>
+          `id` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest" id="identity">id</dfn></code> member is a <a>string</a> that
+          represents the <dfn>identity</dfn> for the application. 
+          The |identity| takes the form of a URL, which is same origin as the
+          start URL.
+        </p>
+        <p>The |identity| is used by user agents to uniquely identify the
+        application universally. When the user agent sees a manifest with an
+        |identity| that does not correspond to an already-installed application,
+        it SHOULD treat that manifest as a description of a distinct
+        application, even if it is served from the same URL as that of another
+        application. When the user agent sees a manifest with an |identity|
+        matching that of an already-installed application, it SHOULD be used as
```

Comment:
We should create a section defining how to tell if they are matching. Because we are parsing as a url, certain things should be considered the same:

* ASCII characters: `j` and `%6a`
* Special: `a😀b` and `a%F0%9F%98%80b`, `a b` and `a%20b`
* Slash? `a/b` would become `a%2Fb`?

I'm not sure the best place to put this. We could have a section like

```
<p>
  Two manifests are considered <dfn  id="matching">matching</dfn> if performing the [=url equivalence=] process on the |identity| members of the manifests returns true.
</p>
```

Perhaps put this above the existing paragraph here, and then you can link |matching|.

Reply:
@marcoscaceres I wasn't able to find the url-equivalence definition in the respec search tool, but the definition is here:
https://url.spec.whatwg.org/#url-equivalence

Not sure how to link to that?

Reply:
Sorry for the delay! we need to ask the editor of that spec to "`<dfn export>`" it. 

I would suggest sending a PR to the URL spec, exporting it and pointing to this pull request explaining why we need it.

Reply:
clarified with using [=url/equal=]